### ISSUE TITLE
[Bug]: Newest version breaks Next.js

### ISSUE THREAD

**Issue by heisenbugespen** [wallabyjs/console-ninja#332]
### What happened?

Buttons stops working and sometimes the whole thing just shows a blank page. Works perfectly fine when disabling Console NInja

### Version

v1.0.346

### CLI command to start your dev tool

npm run dev

### Steps to reproduce (or sample repo)

Start Next.js with npm run dev -> Blank page or buggy buttons

### Log output

```shell
10:44:33.645 info host client connected: extension { address: '127.0.0.1', family: 'IPv4', port: 65336 }
10:44:33.672 info host starting host: {"extensionFolder":"/Users/espensaevig/.vscode/extensions/wallabyjs.console-ninja-1.0.346","runtimeHookFile":"/Users/espensaevig/.vscode/extensions/wallabyjs.console-ninja-1.0.346/out/runtimeHook/tracer.js","globalErrorHandlerFile":"/Users/espensaevig/.vscode/extensions/wallabyjs.console-ninja-1.0.346/out/runtimeHook/errorHandler.js","networkLoggingHandlerFile":"/Users/espensaevig/.vscode/extensions/wallabyjs.console-ninja-1.0.346/out/runtimeHook/networkLoggingHandler.js","filesToInstrument":[{"path":"/users/espensaevig/code/heisenbug/niftyhr/helperfunctions/mail/generateemaildata.tsx","reason":"search"},{"path":"/users/espensaevig/code/heisenbug/niftyhr/helperfunctions/getcompany.tsx","reason":"search"},{"path":"/users/espensaevig/code/heisenbug/niftyhr/components/logingoogle.tsx","reason":"search"},{"path":"/users/espensaevig/code/heisenbug/niftyhr/components/layout/header.tsx","reason":"search"},{"path":"/users/espensaevig/code/heisenbug/niftyhr/components/auth/create/createuser.tsx","reason":"search"},{"path":"/users/espensaevig/code/heisenbug/niftyhr/components/datepicker_mypage.tsx","reason":"search"},{"path":"/users/espensaevig/code/heisenbug/niftyhr/components/forms/bronnoysund.tsx","reason":"search"},{"path":"/users/espensaevig/code/heisenbug/niftyhr/components/vacation_comp/adminapplicationlist.tsx","reason":"search"},{"path":"/users/espensaevig/code/heisenbug/niftyhr/app/page.tsx","reason":"search"},{"path":"/users/espensaevig/code/heisenbug/niftyhr/components/vacation_comp/applicationlist.tsx","reason":"search"},{"path":"/users/espensaevig/code/heisenbug/niftyhr/app/api/email/route.ts","reason":"search"},{"path":"/users/espensaevig/code/heisenbug/niftyhr/app/api/auth/callback/route.ts","reason":"search"},{"path":"/users/espensaevig/code/heisenbug/niftyhr/components/vacation_comp/applicationform.tsx","reason":"search"},{"path":"/users/espensaevig/code/heisenbug/niftyhr/app/dashboard/[orgid]/coworkers/page.tsx","reason":"search"},{"path":"/users/espensaevig/code/heisenbug/niftyhr/app/dashboard/[orgid]/vacation/page.tsx","reason":"search"},{"path":"/users/espensaevig/code/heisenbug/niftyhr/app/auth/login/page.tsx","reason":"search"},{"path":"/users/espensaevig/code/heisenbug/niftyhr/components/vacation_comp/calenderdisplay.tsx","reason":"search"},{"path":"/users/espensaevig/code/heisenbug/niftyhr/middleware.ts","reason":"search"},{"path":"/users/espensaevig/code/heisenbug/niftyhr/components/mypage_comp/profileinfor.tsx","reason":"search"},{"path":"/users/espensaevig/code/heisenbug/niftyhr/components/mypage_comp/inforcard.tsx","reason":"search"},{"path":"/users/espensaevig/code/heisenbug/niftyhr/app/auth/setup/companydetails.tsx","reason":"search"},{"path":"/users/espensaevig/code/heisenbug/niftyhr/app/auth/setup/namedetails.tsx","reason":"search"},{"path":"/users/espensaevig/code/heisenbug/niftyhr/utils/supabase/queries.ts","reason":"search"},{"path":"/Users/espensaevig/Code/heisenbug/niftyhr/app/dashboard/layout.tsx","reason":"open"},{"path":"/Users/espensaevig/Code/heisenbug/niftyhr/app/dashboard/[orgId]/layout.tsx","reason":"open"},{"path":"/Users/espensaevig/Code/heisenbug/niftyhr/components/layout/DashboardMenu.tsx","reason":"open"},{"path":"/Users/espensaevig/Code/heisenbug/niftyhr/components/layout/Sidebar.tsx","reason":"open"},{"path":"/Users/espensaevig/Code/heisenbug/niftyhr/components/layout/CompanySelector.tsx","reason":"open"},{"path":"/Users/espensaevig/Code/heisenbug/niftyhr/app/api/auth/confirm/route.ts","reason":"open"},{"path":"/Users/espensaevig/Code/heisenbug/niftyhr/app/auth/login/page.tsx","reason":"open"},{"path":"/Users/espensaevig/Code/heisenbug/niftyhr/middleware.ts","reason":"open"},{"path":"/Users/espensaevig/Code/heisenbug/niftyhr/components/layout/Header.tsx","reason":"open"}],"debug":false,"config":{"dateTimeFormat":"HH:mm:ss.SSS","dateTimeDisplayMode":"Absolute","installBinToPath":true,"openLinksInEditor":true,"maxLogViewerEntries":15000,"showWelcomeMessageInTools":true,"fontSize":10,"outputMode":"In View","captureFunctions":false,"searchUrl":"https://www.google.com/search?q={query}","enableProFeatures":false,"toolsToAutoPatch":["vite","angular","jest","webpack","next.js","nest.js","cypress","http-server","serve","live-server","nuxt","remix","qwik","hydrogen","serverless","astro","live-server-extension","live-preview-extension"],"showErrors":true,"showLogsOnlyFromOpenedFiles":false,"showLogsFromLogPointsOnly":false,"showLogsFromMultipleRunningToolsTogether":false,"predictiveLoggingEnabled":true,"allowConsoleLogToTriggerPredictions":false,"showNetworkRequests":true},"allowedHosts":["localhost","127.0.0.1","example.cypress.io","Espens-MacBook-Pro.local","10.20.50.93"],"host":"127.0.0.1","forcedStart":false,"startTime":1723711471330}
10:44:33.787 info host starting tools discovery and integration
10:44:33.881 info host looking for tools to patch
10:44:33.892 info host worker Starting Host Worker...
10:44:34.217 info host worker Starting Host Worker...
10:44:34.614 info host found 3 tool file(s) to patch
10:44:34.617 info host patching /users/espensaevig/code/heisenbug/citybox/citybox-app/node_modules/next/dist/compiled/webpack/bundle5.js
10:44:34.618 info host patching /users/espensaevig/code/heisenbug/citybox/citybox-app/node_modules/next/dist/build/webpack/loaders/next-swc-loader.js
10:44:34.778 info host Host's startup and patch time: 3445 ms
10:44:34.779 info host completed tools discovery and integration
10:44:34.786 ERR! host uncaught exception: No clients connected
10:44:34.947 info host found 3 tool file(s) to patch
10:44:34.953 info host patching /users/espensaevig/code/heisenbug/niftyhr/node_modules/next/dist/compiled/webpack/bundle5.js
10:44:34.955 info host patching /users/espensaevig/code/heisenbug/niftyhr/node_modules/next/dist/build/webpack/loaders/next-swc-loader.js
10:44:35.058 info host Host's startup and patch time: 3728 ms
10:44:35.060 info host completed tools discovery and integration
10:44:35.071 ERR! host uncaught exception: No clients connected
10:45:33.066 info host removing cache folder: 19380928a856e1c67b4dce7b83104a39
10:45:34.156 info buildHook-3751 installing build hook for next.js at /users/espensaevig/code/heisenbug/citybox/citybox-app, node v20.9.0
10:45:34.163 info buildHook-3751 allowed tools: vite, angular, jest, webpack, next.js, nest.js, cypress, http-server, serve, live-server, nuxt, remix, qwik, hydrogen, serverless, astro, live-server-extension, live-preview-extension, node
10:45:34.163 info buildHook-3751 running tools: next.js
10:45:34.165 info buildHook-3751 installing fs interceptor
10:45:34.207 info host client connected: buildHook { address: '127.0.0.1', family: 'IPv4', port: 52156 }
10:45:34.236 info buildHook-3751 file processed (stat): 5 log points, error handler: false, network logging handler: false
10:45:34.290 info buildHook-3751 file processed (stat): 36 log points, error handler: false, network logging handler: false
10:45:34.298 info buildHook-3751 file processed (stat): 2 log points, error handler: false, network logging handler: false
10:45:34.301 info buildHook-3751 file processed (stat): 1 log points, error handler: false, network logging handler: false
10:45:34.305 info buildHook-3751 file processed (stat): 1 log points, error handler: false, network logging handler: false
10:45:34.307 info buildHook-3751 file processed (stat): 1 log points, error handler: false, network logging handler: false
10:45:34.319 info buildHook-3751 file processed (stat): 8 log points, error handler: false, network logging handler: false
10:45:34.328 info buildHook-3751 file processed (stat): 3 log points, error handler: false, network logging handler: false
10:45:34.332 info buildHook-3751 file processed (stat): 1 log points, error handler: false, network logging handler: false
10:45:34.334 info buildHook-3751 file processed (stat): 0 log points, error handler: false, network logging handler: false
10:45:34.337 info buildHook-3751 file processed (stat): 0 log points, error handler: false, network logging handler: false
10:45:34.339 info buildHook-3751 file processed (stat): 0 log points, error handler: false, network logging handler: false
10:45:34.340 info buildHook-3751 file processed (stat): 0 log points, error handler: false, network logging handler: false
10:45:34.341 info buildHook-3751 allowed tools: vite, angular, jest, webpack, next.js, nest.js, cypress, http-server, serve, live-server, nuxt, remix, qwik, hydrogen, serverless, astro, live-server-extension, live-preview-extension, node
10:45:34.341 info buildHook-3751 running tools: next.js
10:45:35.143 info buildHook-3751 file processed (stat): 0 log points, error handler: true, network logging handler: true
10:46:01.782 info buildHook-3751 file processed (stat): 0 log points, error handler: true, network logging handler: true
10:46:13.480 info buildHook-3751 file processed (stat): 0 log points, error handler: true, network logging handler: true
```

**Comment by jonas-baek-precis** [comment]
Seeing the same thing - blank page shows with this error in console:
![image](https://github.com/user-attachments/assets/2be0c8d5-b474-4e5e-bf2d-dda821db8d1d)

**Comment by nemanjakodiro** [comment]
I am getting the same issue in v1.0.346 and v1.0.345, v1.0.344 works fine.

**Comment by pronovaso** [comment]
> I am getting the same issue in v1.0.346 and v1.0.345, v1.0.344 works fine.

I can confirm same think

**Comment by Mahdhir** [comment]
Can confirm the same
Had to spend nearly 5 hours trying to figure out the problem

**Comment by mdkamil98** [comment]
Yeah, same thing here, am using next 14

**Comment by ArtemGovorov** [comment]
Thanks for reporting the issue! It's now fixed and the fix is published in v1.0.348.

**Comment by mdkamil98** [comment]
Thanks Man @ArtemGovorov , ur the best
